src/queries/queries.js

```diff
@@ -137,3 +138,27 @@ export const buildOldestTimeseriesQueryByAccountId = accountId => ({
     .sortBy([{ 'cozyMetadata.sourceAccount': 'asc' }, { startDate: 'asc' }])
     .limitBy(1)
 })
+
+export const buildOneYearOldTimeseriesWithAggregationByAccountId = accountId => {
+  const dateOneYearAgoFromNow = startOfMonth(subYears(new Date(), 1))
+
+  return {
+    definition: Q(GEOJSON_DOCTYPE)
+      .where({
+        'cozyMetadata.sourceAccount': accountId,
+        startDate: {
+          $gte: dateOneYearAgoFromNow.toISOString()
+        },
+        aggregation: {
+          $exists: true
+        }
+      })
+      .indexFields(['cozyMetadata.sourceAccount', 'startDate'])
+      .sortBy([{ 'cozyMetadata.sourceAccount': 'asc' }, { startDate: 'asc' }])
+      .limitBy(1000),
```

Comment:
There is no `select` here, so it does not use the benefit of https://github.com/cozy/coachCO2/pull/133 , at all :/
This query will used to fetch the full doc of all the timeseries for the whole year, I think it might make the app freeze in some situations